install-dev/langs/en/data/tab.xml

```diff
@@ -108,7 +108,7 @@
     <tab id="Stock_Coverage" name="Stock Coverage" />
     <tab id="Stock_Management" name="Stock Management" />
     <tab id="Stock_Movement" name="Stock Movement" />
-    <tab id="Stock" name="Stock" />
+    <tab id="Stock" name="Stocks" />
```

Comment:
Here, it's for the menu, according with @superGB products/categories/suppliers/brands/etc/stocks